node/libs/concurrency/src/ctx/mod.rs

```diff
@@ -264,3 +264,28 @@ impl Ctx {
         self.0.rng_provider.rng()
     }
 }
+
+/// anyhow::Error + "canceled" variant.
+/// Useful for working with concurrent code which doesn't need structured errors,
+/// but needs to handle cancelation explicitly.
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
```

Comment:
Bikeshedding: IMO, it would make more sense to put `Error` in the `error` module, near `Wrap`.